tests/test_gpu_examples.py

```diff
@@ -3076,3 +3077,25 @@ def test_bnb_4bit_wrap_fsdp(self):
         init_process_group(world_size=1, rank=0)
         # check that this does not raise:
         FSDP(model, auto_wrap_policy=fsdp_auto_wrap_policy(model), use_orig_params=False, sync_module_states=True)
+
+
+@require_torch_gpu
```

Comment:
This decorator doesn't work as is. The reason why it works on other classes in this file but not here is because these other classes inherit from `unittest.TestCase`. But instead of doing the same, let's just move the decorator on the two test methods directly, then it works as expected.